builtin/providers/packet/config.go

```diff
@@ -0,0 +1,18 @@
+package packet
+
+import (
+	"github.com/packethost/packngo"
+)
+
+const (
+	consumerToken = "aZ9GmqHTPtxevvFq9SK3Pi2yr9YCbRzduCSXF2SNem5sjB91mDq7Th3ZwTtRqMWZ"
+)
+
+type Config struct {
+	AuthToken string
+}
+
+// Client() returns a new client for accessing packet.
+func (c *Config) Client() *packngo.Client {
+	return packngo.NewClient(consumerToken, c.AuthToken)
```

Comment:
I made a note on your other note, happy to add to the docs if that will be clear to end users, but it is not a parameter that should / will change unless folks are editing source code for some reason, like forking terraform for another service or something. does that make sense?